src/Item.h

```diff
@@ -206,6 +222,7 @@ class cItem
 
 	int            m_RepairCost;
 	cFireworkItem  m_FireworkItem;
+	unsigned int m_ItemColor;
```

Comment:
I think if you're going to compress the colour into an int, you should use an object to abstract accessing the various fields.